src/appshell/view/preferences/appearancepreferencesmodel.cpp

```diff
@@ -78,6 +93,48 @@ QStringList AppearancePreferencesModel::accentColors() const
     return uiConfiguration()->possibleAccentColors();
 }
 
+void AppearancePreferencesModel::resetThemeToDefault()
+{
+    uiConfiguration()->resetCurrentThemeToDefault(currentTheme().codeKey);
+    notationConfiguration()->resetCurrentBackgroundColorToDefault();
+    emit backgroundColorChanged();
+    emit themesChanged();
+}
+
+bool AppearancePreferencesModel::enableHighContrastChecked()
+{
+    return uiConfiguration()->isCurrentThemeHighContrast();
+}
+
+void AppearancePreferencesModel::loadLastUsedGeneralTheme()
+{
+    uiConfiguration()->loadLastUsedGeneralTheme();
+}
+
+void AppearancePreferencesModel::loadLastUsedHighContrastTheme()
+{
+    uiConfiguration()->loadLastUsedHighContrastTheme();
+}
+
+void AppearancePreferencesModel::setNewColor(const QColor& newColor, ColorType colorType)
+{
+    switch (colorType) {
+    case AccentColor: uiConfiguration()->setCurrentThemeStyleValue(ThemeStyleKey::ACCENT_COLOR, Val(newColor));
+        break;
+
+    case TextAndIconsColor: uiConfiguration()->setCurrentThemeStyleValue(ThemeStyleKey::FONT_PRIMARY_COLOR, Val(newColor));
+        break;
+
+    case DisabledColor: return;
+
+    case BorderColor: uiConfiguration()->setCurrentThemeStyleValue(ThemeStyleKey::STROKE_COLOR, Val(newColor));
+        break;
+
+    default: qDebug("INVALID COLORTYPE RECEIVED!");
```

Comment:
No need to shout ;-), a normal sentence works too here

Reply:
just remove the default case

Reply:
True, has the added benefit of a compiler warning if new cases get added but not dealt with here

Reply:
Ah, apologies 😅 
Converted to `"Invalid ColorType Received."`
Fixed

Reply:
I'd still say it's even better to remove the `default:` entirely. At the moment, it is not needed, and without it, as Jojo says, the compiler will warn when somebody adds a new case to the enum and forgets to handle that case here.

Reply:
Ah understood
I will remove the default case entirely.